string session_id
string peer_id
---
ros_webrtc/Session session
